src/Momento.Sdk/Internal/ScsDataClient.cs

```diff
@@ -16,16 +14,16 @@ namespace Momento.Sdk.Internal;
 public class ScsDataClientBase : IDisposable
 {
     internal readonly DataGrpcManager grpcManager;
-    private readonly uint defaultTtlSeconds;
-    private readonly uint dataClientOperationTimeoutMilliseconds;
-    private readonly ILogger _logger;
+    protected readonly int defaultTtlMilliseconds;
+    protected readonly int dataClientOperationTimeoutMilliseconds;
+    protected readonly ILogger _logger;
```

Comment:
curious why the increase in visibility on these

Reply:
That got clobbered in the several merges and got through in spite of looking this over several times. Thank you. Fixed in 7e1af1e.